embedded/sql/row_reader.go

```diff
@@ -99,17 +107,27 @@ func (row *Row) digest(cols []ColDescriptor) (d [sha256.Size]byte, err error) {
 }
 
 type rawRowReader struct {
-	tx              *SQLTx
-	table           *Table
-	asBefore        uint64
-	tableAlias      string
-	colsByPos       []ColDescriptor
-	colsBySel       map[string]ColDescriptor
-	scanSpecs       *ScanSpecs
+	tx         *SQLTx
+	table      *Table
+	tableAlias string
+	colsByPos  []ColDescriptor
+	colsBySel  map[string]ColDescriptor
+	scanSpecs  *ScanSpecs
+
+	txRange *txRange
```

Comment:
txRange is initialized from period, correct? Maybe would be good to add some small comment about that?